app/handlers/create_password.rb

```diff
@@ -1,26 +1,34 @@
+# Changes the user's password if they have one, otherwise creates a password (an `Identity`).
 class CreatePassword
   lev_handler
-  uses_routine ::SetPassword
+  uses_routine SetPassword
```

Comment:
why remove `::` from here but not line 23? Does `::` do something special?

Reply:
The `::` ensures the SetPassword being called is the top-level one, not SomeModule::SetPassword.
It's not necessary unless you have multiple SetPassword or think you may have them in the future. I can probably remove all of them.